news/_posts/2020-04-27-Polymake.jl-0.4-has-been-tagged.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Polymake.jl 0.4 has been tagged
+author: Marek Kaluba
+---
+
+`Polymake.jl` v0.4.0 has been tagged!
+
+Since we're upgrading to support `CxxWrap.jl` `v0.10.1`, we decided to bump the major 
```

Comment:
```suggestion
Since we're upgrading to support `CxxWrap.jl` v0.10.1, we decided to bump the major 
```
The version is not code, so no need to put it into backticks. It's fine to do use backticks for package names (of course that's a stylistic matter in the end)